presto-raptor/src/test/java/com/facebook/presto/raptor/integration/TestRaptorIntegrationSmokeTest.java

```diff
@@ -632,6 +636,8 @@ public void testShowCreateTable()
         actualResult = computeActual("SHOW CREATE TABLE " + getSession().getCatalog().get() + "." + getSession().getSchema().get() + ".test_show_create_table_organized");
         assertEquals(getOnlyElement(actualResult.getOnlyColumnAsSet()), createTableSql);
 
+        // For table_supports_delta_delete, we don't have null value, thus can't distinguish between user set false and default false
```

Comment:
hmmm, why don't we have nulls then? The default is also false if not specified.

Reply:
This is by design.
@jessesleeping do you have any thoughts about James's comment?